### ISSUE TITLE
Errors when compiling /example with latest packages

### ISSUE THREAD

**Issue by dekryptic** [bcherny/json-schema-to-typescript#134]
With this package.json under example/:
```
{
  "name": "json-schema-to-typescript-example",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "typings": "index.d.ts",
  "scripts": {
    "build": "npm run clean && tsc && node index.js",
    "clean": "rm -f index.js index.d.ts person.d.ts"
  },
  "dependencies": {
    "@types/node": "^8.0.58",
    "typescript": "^2.6.2",
    "json-schema-to-typescript": "^5.2.0"
  }
}
```

I'm getting these errors:
```
node_modules/json-schema-to-typescript/dist/src/index.d.ts(1,29): error TS2307: Cannot find module 'json-schema'.
node_modules/json-schema-to-typescript/dist/src/index.d.ts(2,44): error TS7016: Could not find a declaration file for module 'prettier'. '/.../json-schema-to-typescript/example/node_modules/prettier/index.js' implicitly has an 'any' type.
  Try `npm install @types/prettier` if it exists or add a new declaration (.d.ts) file containing `declare module 'prettier';`
node_modules/json-schema-to-typescript/dist/src/types/JSONSchema.d.ts(1,29): error TS2307: Cannot find module 'json-schema'.
```

**Comment by bcherny** [comment]
Thanks for the report @dekryptic!

Fix is published in `+ json-schema-to-typescript@5.2.1`

**Comment by dekryptic** [comment]
5.2.2 worked for me, while 5.2.1 didn't. 

Can you explain how you fixed the issue? I'm rather new to Typescript and just want to learn.

**Comment by bcherny** [comment]
So the actual fix was in 5.2.2 - 5.2.1 was a red herring.

The thing is, TypeScript requires that all `import`s have [corresponding typings](https://www.typescriptlang.org/docs/handbook/modules.html). We install those typings by running `npm install @types/abc`, replacing `abc` with the package name.

JSON-Schema-to-TS depends on the `json-schema` and `prettier` packages, and also installs typings for them. Those typings are listed in the package.json, but they're listed under `devDependencies`, not `dependencies`. This means that when you install JSON-Schema-to-TS as a dependency for your project, the typings aren't installed! I assume most people use JSON-Schema-to-TS's CLI or via JavaScript, because if someone used JSON-Schema-to-TS programmatically via TypeScript they would get this same error.

The fix was to move the typings from `devDependencies` to `dependencies`, so they are installed when you install JSON-Schema-to-TS as a dependency.

Hopefully that makes sense - let me know if I can clarify anything!
